nautobot/core/views/routers.py

```diff
@@ -79,4 +79,22 @@ class NautobotUIViewSetRouter(SimpleRouter):
             detail=True,
             initkwargs={"suffix": "Edit"},
         ),
+        Route(
+            url=r"^{prefix}/{lookup}/changelog/$",
+            mapping={
+                "get": "changelog",
+            },
+            name="{basename}_changelog",
+            detail=True,
+            initkwargs={"suffix": "Changelog"},
+        ),
+        Route(
+            url=r"^{prefix}/{lookup}/notes/$",
+            mapping={
+                "get": "notes",
+            },
+            name="{basename}_notes",
+            detail=True,
+            initkwargs={"suffix": "Notes"},
+        ),
```

Comment:
Does this pose any issue for existing plugins that are using the 1.4 version of `NautobotUIViewSetRouter` and separately implementing the changelog and notes URL patterns as was previously needed? Want to make sure we're not doing something backwards-incompatible.

Reply:
I just tested with Providers (by adding back in the path to urls.py, and it doesn't break anything. I am not sure which view it is loading... But nothing broke.